src/pages/LoginPage/LoginPage.css

```diff
@@ -116,3 +114,34 @@
     width: 100%;
   }
 }
+.password-wrapper {
+  display: flex;
+  align-items: center;
+  height: 2.5rem;
```

Comment:
remove this as it makes the password input smaller compared to the other inputs.